src/fqdncache.cc

```diff
@@ -172,7 +174,7 @@ fqdncache_get(const char *name)
     f = NULL;
 
     if (fqdn_table) {
-        if ((e = (hash_link *)hash_lookup(fqdn_table, name)) != NULL)
+        if ((e = (hash_link *)fqdn_table->hash_lookup(name)) != nullptr)
```

Comment:
This should not need any null comparison at all now.
```suggestion
        if ((e = (hash_link *)fqdn_table->hash_lookup(name)))
```